src/cook-web/src/app/c/[[...id]]/Components/ChatUi/AskBlock.module.css

```
@@ -132,7 +132,10 @@
   background: #fff;
   border-radius: 24px 24px 0 0;
   box-shadow: 0 -12px 32px rgba(28, 44, 64, 0.16);
+  box-sizing: border-box;
+  padding-bottom: var(--keyboard-offset, 0px);
   max-height: 85vh;
+  max-height: min(85vh, var(--app-height, 100vh));
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

max-height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)